cadence/transactions/evm/create_new_account_with_coa.cdc

```diff
@@ -0,0 +1,60 @@
+import Crypto
+
+import "EVM"
+
+/// Creates a new Flow Address with a single full-weight key and its EVM account, which is
+/// a Cadence Owned Account (COA) stored in the account's storage.
+///
+transaction(
+    key: String,  // key to be used for the account
+    signatureAlgorithm: UInt8, // signature algorithm to be used for the account
+    hashAlgorithm: UInt8, // hash algorithm to be used for the account
+) {
+    let auth: auth(Storage, Keys, Capabilities) &Account
+
+    prepare(signer: auth(Storage, Keys, Capabilities) &Account) {
```

Comment:
All we really need in the `execute` block is `BorrowValue`. It might be better from the signer's perspective to instead share the `&Account` reference on the created account (creating the account in the `prepare` block) since the extra authorizations are only needed on the created account. 

```suggestion
    let auth: auth(BorrowValue) &Account

    prepare(signer: auth(BorrowValue) &Account) {
```